tools/code-owners-parser/CodeOwnersParser/CodeOwnersParser.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Not sure it matters but we might want to make this netstandard2.0 so we could consume this in other target frameworks if needed.